modules/users/server/config/strategies/facebook.js

```diff
@@ -5,9 +5,10 @@
  */
 var passport = require('passport'),
 	FacebookStrategy = require('passport-facebook').Strategy,
-	users = require('../../controllers/users.server.controller');
+	User = require('mongoose').model('User'),
+	users = require('./../../controllers/users.server.controller');
```

Comment:
It's pretty clear for me, but adding a resolve doesn't make much sense when it's just a one time thing and you can check the file tree in Sublime/Atom etc. to verify it.